package.json

```diff
@@ -23,6 +24,20 @@
     "babel-preset-es2015": "^6.6.0",
     "babel-preset-react": "^6.5.0",
     "babel-preset-stage-0": "^6.5.0",
+    "bootstrap": "^3.3.7",
```

Comment:
fair point! I'm not sure if that is a common practice, to be honest
on production environment I'd go for running integration/smoke tests that treat the application as a black box
but if they treat it as a black box and use public api, it doesn't matter where they're run, they just need access to a running app
also, we don't have any tests here which is a whole other story